changelog/unreleased/enhancement-change-move-navigation

```diff
@@ -0,0 +1,8 @@
+Change: Move OcSidebarNav and OcSidebarNavItem to web
+
+We've moved OcSidebarNav and OcSidebarNavItem to web and renamed it
```

Comment:
Nono. Must not be a one-word component name (reserved for html). Didn't you already change that in web?

Reply:
yes i did change it in web already ^^  i'm not sure what wrong about the changelog 😅 ?

Reply:
I think the changelog item name 😬

Reply:
the part with `renamed it to Navigation and NavigationItem`. I just have the comment on the wrong line of the changelog, sorry. 😅  The component names in web are `SidebarNav` and `SidebarNavItem` - which is good. The changelog item here just references them wrong.

Reply:
But yes, the changelog item name is wrong as well as @pascalwengerter pointed out 😅 instead of `enhancement-change-move-navigation` it should just be `change-move-navigation`.

Reply:
ok will fix it